cdap-app-templates/cdap-dq/src/main/java/co/cask/cdap/dq/AggregationTypeValue.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.dq;
+
+import java.util.Objects;
+
+/**
+ * Class for the object that will represent a given aggregation type. For now, this class only has a field
+ * for the aggregationTypeName
+ */
+public class AggregationTypeValue {
+  private final String name;
+  private boolean isCombinable;
+
+  public AggregationTypeValue(String name, boolean isCombinable) {
+    this.name = name;
+    this.isCombinable = isCombinable;
+  }
+
+  public String getAggregationTypeName() {
+    return name;
+  }
+
+  public boolean isCombinable() {
+    return isCombinable;
+  }
+
+  @Override
+  public boolean equals(Object obj) {
+    if (!(obj instanceof AggregationTypeValue)) {
+      return false;
+    }
+    if (obj == this) {
+      return true;
+    }
+    AggregationTypeValue agTypeListObj = (AggregationTypeValue) obj;
+    return name.equals(agTypeListObj.name) && (isCombinable == agTypeListObj.isCombinable);
```

Comment:
was originally there, but better to use Objects.equals for the string comparison